batchflow/tests/dataformat_test.py

```diff
@@ -7,10 +7,10 @@
 
 from batchflow import Pipeline, ImagesBatch, Dataset
 from batchflow import B, V, C
-from batchflow.models.tf import VGG7, ResNet18, Inception_v1, MobileNet
+from batchflow.models.tf import VGG7, ResNet18, Inception_v1, MobileNet, MobileNet_v2, MobileNet_v3, MobileNet_v3_small
 
 
-MODELS = [VGG7, ResNet18, Inception_v1, MobileNet]
+MODELS = [VGG7, ResNet18, Inception_v1, MobileNet, MobileNet_v2, MobileNet_v3, MobileNet_v3_small]
```

Comment:
What is the time of running the tests on each of these models?
If the time is reasonable to wait, we can add these architectures to `notebooks_tests`.